ports/stm/boards/feather_stm32f405_express/mpconfigboard.mk

```diff
@@ -16,3 +16,5 @@ LD_DEFAULT = boards/STM32F405_default.ld
 # UF2 boot option
 LD_BOOT = boards/STM32F405_boot.ld
 UF2_OFFSET = 0x8010000
+
+CIRCUITPY_CANIO = 1
```

Comment:
Do you want to enable this on all F4s instead?

Reply:
CAN does not exist on Access tier F4s, and we don't support any foundation line ones, so there's not much point in doing so at the moment. It could be moved to mpconfigport under an `#ifdef STM32F405` though.

Reply:
I changed the enable to be in mpconfigboard.mk ifeq ($(MCU_VARIANT),STM32F405xx)